src/hyperlight_guest/third_party/musl/include/setjmp.h

```diff
@@ -0,0 +1,49 @@
+#ifndef	_SETJMP_H
```

Comment:
What was the reason @syntactically removed the musl setjmp before? Are we fine to bring this back?